package.json

```diff
@@ -14,9 +14,9 @@
     "lint": "npm i && npm i --legacy-peer-deps --prefix ./src/ui && eslint --ext .ts src/ui && npm i --prefix ./src/api && eslint --ext .ts src/api && eslint --ext .js e2e-tests",
     "lint:fix": "prettier --write src/ui  && eslint --ext .ts src/api --fix && eslint --ext .ts src/ui --fix &&  eslint --ext .js e2e-tests --fix",
     "prepare": "husky install",
-    "install:all": "npm i && npm i --legacy-peer-deps --prefix ./src/ui && npm i --legacy-peer-deps --prefix ./src/api && npm i --legacy-peer-deps --prefix ./src/generate-exip-pricing-grid",
-    "update:all": "npm update && npm update --legacy-peer-deps --prefix ./src/ui && npm update --prefix ./src/api && npm update --prefix ./src/generate-exip-pricing-grid",
-    "audit:all": "npm audit && npm audit --prefix ./src/ui && npm audit --prefix ./src/api && npm audit --prefix ./src/generate-exip-pricing-grid"
+    "install:all": "npm i && npm i --legacy-peer-deps --prefix ./src/ui && npm i --legacy-peer-deps --prefix ./src/api && npm i --legacy-peer-deps --prefix ./generate-exip-pricing-grid",
+    "update:all": "npm update && npm update --legacy-peer-deps --prefix ./src/ui && npm update --prefix ./src/api && npm update --prefix ./generate-exip-pricing-grid",
```

Comment:
Seems that `generate-exip-pricing-grid` should have `/src/`?

Reply:
Good spot, have removed.